deployment/app.yml

```diff
@@ -90,6 +90,10 @@ spec:
               value: NEW_RELIC_LICENSE_KEY_STUB
             - name: NEW_RELIC_APP_NAME
               value: NEW_RELIC_APP_NAME_STUB
+            - name: ENVIRONMENT
```

Comment:
issue: now that I'm looking at the full file, I think we will have a problem because ENVIRONMENT is being replaced at the top. If the regex is applied globally, this will mess up with the ENVIRONMENT_STUB